src/components/flow/flow.tsx

```diff
@@ -120,33 +98,13 @@ export class Flow {
   };
 
   handleMutationObserverChange = (): void => {
-    const { el, allowDescendantPanels } = this;
-
-    if (!allowDescendantPanels) {
-      return;
-    }
-
     const newPanels: HTMLCalcitePanelElement[] = Array.from(
-      el.querySelectorAll("calcite-panel:not(calcite-panel calcite-panel)")
+      this.el.querySelectorAll("calcite-panel:not(calcite-panel calcite-panel)")
```

Comment:
Will this query still work if you have a 3rd nested `calcite-panel` like `(calcite-panel calcite-panel calcite-panel)` or even beyond that?

Reply:
Wouldn't that fall under `:not(calcite-panel calcite-panel)`?